docs/source/getting_started/using_entitysets.rst

```diff
@@ -63,7 +66,7 @@ This method loads each column in the dataframe in as a variable. We can see the
 
     es["transactions"].variables
 
-In the call to ``entity_from_dataframe``, we specified three important parameters
+In the call to ``add_dataframe``, we specified three important parameters
 
 * The ``index`` parameter specifies the column that uniquely identifies rows in the dataframe
 * The ``time_index`` parameter tells Featuretools when the data was created.
```

Comment:
Since we changed the call above to use `logical_types` we should update the line below this as well to change `variable_types` to `logical_types`.

Reply:
updated